<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KoStrokeConfigWidget</class>
 <widget class="QWidget" name="KoStrokeConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="fillConfigWidgetLayout"/>
   </item>
   <item>
    <widget class="Line" name="thicknessLineBreak">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thicknessLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="thicknessLabel">
       <property name="text">
        <string>Thickness:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisDoubleParseUnitSpinBox" name="lineWidth" native="true"/>
     </item>
     <item>
      <widget class="QToolButton" name="capNJoinButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lineStyleLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="strokeStyleLabel">
       <property name="text">
        <string>Line Style:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KoLineStyleSelector" name="lineStyle" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="markerLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleParseUnitSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_double_parse_unit_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>KoLineStyleSelector</class>
   <extends>QWidget</extends>
   <header>KoLineStyleSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>